### DISCUSSION TITLE
How can i get value of get_authentication_url inside template>

### DISCUSSION THREAD

**Discussion by britanik** [oidc-wp/openid-connect-generic#311] [Q&A] [answered]
I want to create completely custom functionality around auth link and want to be able to get value of get_authentication_url inside template. How can i do that?

Thanks for your help!

**Comment by timnolte** [comment, marked as answer]
You can use the `openid_connect_generic_auth_url` shortcode in your template.

**Reply by britanik** [reply]
This worked:
$url = do_shortcode('[openid_connect_generic_auth_url]');

Thanks for your help!
